src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -20,104 +24,81 @@ class RegistrationServiceImplTest {
     static void beforeAll() {
         storageDao = new StorageDaoImpl();
         registrationService = new RegistrationServiceImpl();
-
     }
 
     @BeforeEach
     void beforeEach() {
         user = new User();
+        user.setLogin(DEFAULT_LOGIN);
+        user.setAge(DEFAULT_AGE);
+        user.setPassword(DEFAULT_PASSWORD);
     }
 
     @Test
     void register_userLoginOnNull_notOk() {
-        user.setAge(23);
-        user.setPassword("qwer1234");
+        user.setLogin(null);
         Exception exception = assertThrows(InvalidUserDataException.class,
-                () -> registrationService.register(user));
-        assertEquals("Login can't be empty!", exception.getMessage());
+                () -> registrationService.register(user), "Login can't be empty!");
     }
 
     @Test
     void register_userLoginIsEmpty_notOk() {
         user.setLogin("");
-        user.setAge(23);
-        user.setPassword("qwer1234");
         Exception exception = assertThrows(InvalidUserDataException.class,
-                () -> registrationService.register(user));
-        assertEquals("Login can't be empty!", exception.getMessage());
+                () -> registrationService.register(user), "Login can't be empty!");
     }
 
     @Test
     void register_userLoginHasTheSameLogin_notOk() {
-        user.setLogin("dima");
-        user.setAge(23);
-        user.setPassword("qwer1234");
         storageDao.add(user);
         User user1 = new User();
-        user1.setLogin("dima");
+        user1.setLogin(DEFAULT_LOGIN);
         user1.setAge(32);
         user1.setPassword("qws4123123");
         Exception exception = assertThrows(InvalidUserDataException.class,
-                () -> registrationService.register(user1));
-        assertEquals("User with this login "
-                + user1.getLogin() + " is already registered!", exception.getMessage());
+                () -> registrationService.register(user1), "User with this login "
+                        + user1.getLogin() + " is already registered!");
```

Comment:
try to add the same user one more time without creating a new one and try to catch an error at the same time